src/dicomweb_client/api.py

```diff
@@ -1745,6 +1783,30 @@ def retrieve_study_metadata(
         url += '/metadata'
         return self._http_get_application_json(url)
 
+    def delete_study(self, study_instance_uid: str) -> None:
+        '''Deletes specified study and its respective instances.
+
+        Parameters
+        ----------
+        study_instance_uid: str
+            unique study identifier
+
+        Returns
+        -------
+        requests.models.Response
+            HTTP response object returned.
+
```

Comment:
Include `Note` that this is not a standard method:

```suggestion
        Note
        ----
        The Delete Study resource is not part of the DICOM standard
        and may not be supported by all origin servers.
```